### DISCUSSION TITLE
community-container idea: owntrack

### DISCUSSION THREAD

**Discussion by jakobroehrl** [nextcloud/all-in-one#5239] [Ideas] [answered]
https://owntracks.org/

**Comment by szaimen** [comment, marked as answer]
Hi, interesting idea but why don't you use this app? https://apps.nextcloud.com/apps/phonetrack

Anyway, I found this documentation on the project: https://github.com/owntracks/docker-recorder/?tab=readme-ov-file#with-mqtt-broker. [The community container feature](https://github.com/nextcloud/all-in-one/tree/main/community-containers#community-containers) could be used for that. See [this doc](https://github.com/nextcloud/all-in-one/tree/main/community-containers#how-to-add-containers) how to add a container to the list.

**Reply by szaimen** [reply]
@jakobroehrl would you be up for adding it since you seem to be interested in the project?

**Comment by jakobroehrl** [comment]
I look over it, if there is some time
